internal/util/update/update_test.go

```diff
@@ -678,6 +678,46 @@ func TestCommand_Run_failInvalidPath(t *testing.T) {
 	}
 }
 
+// TestCommand_Run_failInvalidRepo verifies Run fails if the repo is invalid
+func TestCommand_Run_failInvalidRepo(t *testing.T) {
+	for _, strategy := range kptfilev1alpha2.UpdateStrategies {
+		t.Run(string(strategy), func(t *testing.T) {
+			g := &testutil.TestSetupManager{
+				T: t,
+				ReposChanges: map[string][]testutil.Content{
+					testutil.Upstream: {
+						{
+							Data:   testutil.Dataset1,
+							Branch: "master",
+						},
+						{
+							Data: testutil.Dataset2,
+						},
+					},
+				},
+			}
+			defer g.Clean()
+			if !g.Init() {
+				return
+			}
+
+			err := Command{
+				Pkg:      pkgtest.CreatePkgOrFail(t, g.LocalWorkspace.FullPackagePath()),
+				Repo:     "fake",
+				Strategy: strategy,
+			}.Run()
+			if !assert.Error(t, err) {
+				return
```

Comment:
nit: We usually use `t.FailNow()` in these situations.

Reply:
Replaced.